File: TelerikUIExamplesInCSharp/Examples/DataSource/DSItem.cs
using System;

using Foundation;
using UIKit;

namespace Examples
{
	public class DSItem: NSObject
	{
		[Export("Name")]
		public string Name { get; set; }

		[Export("Value")]
		public nfloat Value { get; set; }

		[Export("Group")]
		public string Group { get; set; }

		[Export("Date")]
		public NSDate Date { get; set; }

		[Export("Image")]
		public UIImage Image { get; set; }
	}
}

File: TelerikUIExamplesInSwift2.0/TelerikUIExamplesInSwift/DSItem.swift
//
//  DSItem.swift
//  TelerikUIExamplesInSwift
//
//  Copyright (c) 2015 Telerik. All rights reserved.
//

import UIKit

class DSItem: NSObject {
    
    var name: String = ""
    
    var value: Float = 0
    
    var group: String = ""
    
    var image: UIImage?
    
    var date: NSDate?
    
    override init() {
        super.init()
    }
    
    init(name: String, value: Float, group: String) {
        self.name = name
        self.value = value
        self.group = group
    }
}
